log10/load.py

```
@@ -101,31 +101,33 @@ def get_session_id():
     if target_service == "bigquery":
         return str(uuid.uuid4())
 
+    session_id = None
     try:
         res = post_session_request()
-
-        return res.json()["sessionID"]
+        session_id = res.json()["sessionID"]
     except requests.HTTPError as http_err:
         if "401" in str(http_err):
-            raise Exception(
+            logging.warn(
```

We could raise here, since the call site suppress the exception